### PR TITLE
Fix poco test

### PR DESCRIPTION
**Pull request by ziyue-101**
### Change description

Fix for error
```
=== NAME  TestCreateNoChangeUpdateDelete/gkehub/basic-basicpocogkehubfeaturemembership
    dynamic_controller_integration_test.go:234: reconcile for GKEHubFeatureMembership:spo7ttueh7h4kei/gkehubfeaturemembership-basic-poco-spo7ttueh7h4kei took 2.541537647s, result was ({false 0s}, Update call failed: error applying desired state: googleapi: Error 400: InvalidValueError for field membership_specs["projects/xxxx/locations/us-central1/memberships/gkehubmembership-basic-poco-spo7ttueh7h4kei"].feature_spec: does not match a current membership in this project. Keys should be in the form: projects/xxxx/locations/{l}/memberships/{m})
    reconcile.go:191: error was not considered transient; chain is [[*fmt.wrapError: Update call failed: error applying desired state: googleapi: Error 400: InvalidValueError for field membership_specs["projects/xxx/locations/us-central1/memberships/gkehubmembership-basic-poco-spo7ttueh7h4kei"].feature_spec: does not match a current membership in this project. Keys should be in the form: projects/xxx/locations/{l}/memberships/{m}] [*fmt.wrapError: error applying desired state: googleapi: Error 400: InvalidValueError for field membership_specs["projects/xxxx/locations/us-central1/memberships/gkehubmembership-basic-poco-spo7ttueh7h4kei"].feature_spec: does not match a current membership in this project. Keys should be in the form: projects/xxxxx/locations/{l}/memberships/{m}] [*googleapi.Error: googleapi: Error 400: InvalidValueError for field membership_specs["projects/xxx/locations/us-central1/memberships/gkehubmembership-basic-poco-spo7ttueh7h4kei"].feature_spec: does not match a current membership in this project. Keys should be in the form: projects/xxxx/locations/{l}/memberships/{m}]]
```

### Tests you have done

- [ ] Run `make ready-pr` to ensure this PR is ready for review.
- [ ] Perform necessary E2E testing for changed resources.

### CODE REVIEW

File: pkg/test/resourcefixture/testdata/basic/gkehub/v1beta1/gkehubfeaturemembership/basicpocogkehubfeaturemembership/_generated_object_basicpocogkehubfeaturemembership.golden.yaml [hunk 1]
```diff
@@ -15,7 +15,7 @@ spec:
   featureRef:
     name: gkehubfeature-basic-poco-${uniqueId}
   location: global
-  membershipLocation: us-central1
+  membershipLocation: us-west1
```
Comment (acpana) [inline pkg/test/resourcefixture/testdata/basic/gkehub/v1beta1/gkehubfeaturemembership/basicpocogkehubfeaturemembership/_generated_object_basicpocogkehubfeaturemembership.golden.yaml:18]: so the issue here was we were using the wrong region in the post submit setup?

**Review by acpana — COMMENTED** [review]
/lgtm
/approve

pending green postsubmit run!

File: pkg/test/resourcefixture/testdata/basic/gkehub/v1beta1/gkehubfeaturemembership/basicpocogkehubfeaturemembership/_generated_object_basicpocogkehubfeaturemembership.golden.yaml (hunk 1 above)
Comment (ziyue-101) [reply pkg/test/resourcefixture/testdata/basic/gkehub/v1beta1/gkehubfeaturemembership/basicpocogkehubfeaturemembership/_generated_object_basicpocogkehubfeaturemembership.golden.yaml:18]: yes

**Review by maqiuyujoyce — COMMENTED** [review]
/approve

File: pkg/test/resourcefixture/testdata/basic/gkehub/v1beta1/gkehubfeaturemembership/basicpocogkehubfeaturemembership/_generated_object_basicpocogkehubfeaturemembership.golden.yaml (hunk 1 above)
Comment (acpana) [reply pkg/test/resourcefixture/testdata/basic/gkehub/v1beta1/gkehubfeaturemembership/basicpocogkehubfeaturemembership/_generated_object_basicpocogkehubfeaturemembership.golden.yaml:18]: We should probably add this to the mock too in order to mimic the behavior here but that can be done later! Let's just not forget about it? We can put it in an issue.

**Pull request merged**